Deployments/suex-enginfprojects-test/k8s-manifests/ingress-default.yml

```
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: enginfprojects-ingress-default
+  namespace: suex-enginfprojects-test
+  annotations:
```

```suggestion
  annotations:
    kubernetes.io/tls-acme: "true"
```
Using Let's encrypt. Certificate is stored in the TLS secret and reused by the celeray-scheduler ingress obejct.